django_fido/views.py

```diff
@@ -99,7 +108,11 @@ def get(self, request: HttpRequest) -> JsonResponse:
         try:
             request_data, state = self.create_fido2_request()
         except ValueError as error:
-            return JsonResponse({'error': force_text(error)}, status=BAD_REQUEST)
+            return JsonResponse({
+                'error_code': getattr(error, 'error_code', None),
```

Comment:
I'd set `Fido2ServerError` as a default `error_code`. At least to split errors from server and client.

Reply:
Fixed.